src/main/java/Main.java

```diff
@@ -0,0 +1,64 @@
+import java.io.File;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.CsvFileReaderService;
+import service.CsvFileWriterService;
+import service.FileReaderService;
+import service.FileWriterService;
+import service.FruitTransactionConverter;
+import service.FruitTransactionConverterImpl;
+import service.Printer;
+import service.PrinterImpl;
+import service.ReportService;
+import service.ReportServiceImpl;
+import service.TransactionStrategy;
+import service.TransactionStrategyImpl;
+import service.transaction.BalanceTransactionHandler;
+import service.transaction.PurchaseTransactionHandler;
+import service.transaction.ReturnTransactionHandler;
+import service.transaction.SupplyTransactionHandler;
+import service.transaction.TransactionHandler;
+
+public class Main {
+    public static final Map<FruitTransaction.Operation,
+                        TransactionHandler> transactionHandlerMap = new HashMap<>();
+    public static final TransactionHandler balanceTransactionHandler =
+                                        new BalanceTransactionHandler();
+    public static final TransactionHandler purchaseTransactionHandler =
+                                        new PurchaseTransactionHandler();
+    public static final TransactionHandler returnTransactionHandler =
+                                        new ReturnTransactionHandler();
+    public static final TransactionHandler supplyTransactionHandler =
+                                        new SupplyTransactionHandler();

```

Comment:
can't see real reason to extract handlers to class variables, they used only once, you may just inline new ProperTransactionHandler() when initializing handlerMap